terraform/main.tf

```diff
@@ -68,3 +68,16 @@ module "gh_oidc" {
     }
   }
 }
+
+resource "google_service_account" "service_account" {
+  account_id   = "rmi-beta-access"
+  display_name = "RMI Beta Access"
+}
+
+resource "google_storage_bucket_iam_binding" "binding" {
```

Comment:
Chose `binding` instead of `policy` so we don't accidentally lock the rest of us out of the `builds.catalyst.coop` bucket. Though eventually we should pull those permissions *into* terraform too, seemed out of scope for this small RMI request.